GoTeamApp/GoTeamApp/SelectedLocationsManager.swift

```diff
@@ -31,6 +31,15 @@ class SelectedLocationsManager {
     }
   }
   
+  func update(location: Location) {
+    queue.async {
+      self.locations = self.locations.filter() { $0 == location }
```

Comment:
Thanks for your comments. Here is what I did. It seems to work fine.
      // update location with new values
      if let index = selectedLocationIndex {
        let location = selectedLocationsManager.locations[index]
        location.title = updatedPin.title
        location.subtitle = updatedPin.subtitle
        selectedLocationsManager.locations[index] = location
        selectedLocationsManager.update(location: location)
      }